sql/core/benchmarks/DateTimeRebaseBenchmark-jdk11-results.txt

```diff
@@ -26,17 +15,6 @@ before 1582, vec off, rebase on                   13836          13872
 before 1582, vec on, rebase off                    3664           3672           7         27.3          36.6       3.5X
 before 1582, vec on, rebase on                     6049           6078          30         16.5          60.5       2.1X
 
-OpenJDK 64-Bit Server VM 11.0.6+10-post-Ubuntu-1ubuntu118.04.1 on Linux 4.15.0-1063-aws
-Intel(R) Xeon(R) CPU E5-2670 v2 @ 2.50GHz
-Save timestamps to parquet:               Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
```

Comment:
Why do you delete it? How should I know the performance in end-to-end case in write?

Reply:
If you decide to merge this PR, I would ask to do that after the optimizations https://github.com/apache/spark/pull/28119 otherwise I will not have good base line for end-to-end performance in write.

Reply:
Yea we can wait for that. IIUC the optimization can also benefit rebasing when writing ORC files, right?

Reply:
days rebasing in ORC will do after the optimization because it uses DaysWritable which is based on functions touched in this PR

Reply:
timestamps rebasing in ORC won't get any benefits from the optimization because it uses to/fromJavaTimestamp

Reply:
can't `toJavaTimestamp` benefit from your optimization?

Reply:
No, it does not use rebaseGregorianToJulianMicros, see https://github.com/apache/spark/blob/820bb9985a76a567b79ffb01bbd2d32788e1dba0/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/DateTimeUtils.scala#L149-L152 . We need to migrate it on the function as I am going to do for days rebasing in https://github.com/apache/spark/pull/28091